SetupDataPkg/SetupDataDxe/SchemaXmlHashDriver/SchemaXmlHash.inf

```diff
@@ -0,0 +1,32 @@
+## @file SchemaXmlHash.inf
+#  Set config xml hash to BIOS variable
+#
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#
+##
+
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = SchemaXmlHashDxe
+  FILE_GUID                      = EFAC7A4E-FB90-450B-9719-349CFFC92487
+  MODULE_TYPE                    = DXE_DRIVER
+  VERSION_STRING                 = 1.0
+  ENTRY_POINT                    = SchemaXmlHashDxeEntry
+
+[Sources]
+  SchemaXmlHash.c
+
+[Packages]
+  MdePkg/MdePkg.dec
+  PlatformPkg/PlatformPkg.dec
+  SetupDataPkg/SetupDataPkg.dec
+
+[LibraryClasses]
+  BaseLib
+  UefiDriverEntryPoint
+  DebugLib
+  PcdLib

```

Comment:
PcdLib isn't used.

UefiRuntimeServicesTable is used but not listed here.

Reply:
@kanechen66, can you also sort the library class names? There is no reason not to and it makes finding duplicate entries easier in the future.

Reply:
done
https://github.com/microsoft/mu_feature_config/commit/05d3f945944ae7b5c161faaba1792f9cc9d54c6c#diff-7529a2b06390ac8534f189e51ac5e73cb623d24b4396fa81a8a6a5fb243eaf5aR30